src/hotspot/cpu/aarch64/templateTable_aarch64.cpp

```diff
@@ -2484,7 +2550,8 @@ void TemplateTable::getfield_or_static(int byte_no, bool is_static, RewriteContr
   // the stores in one method and we interpret the loads in another.
   if (!CompilerConfig::is_c1_or_interpreter_only_no_jvmci()){
     Label notVolatile;
-    __ tbz(raw_flags, ConstantPoolCacheEntry::is_volatile_shift, notVolatile);
+    __ andr(raw_flags, raw_flags, 0x1);
```

Comment:
It looks like `andr` is not needed here?

Reply:
It looks like I misunderstood what `tbz` does! I believe you are correct in suggesting that the `andr` is not necessary.